ocs_ci/helpers/dr_helpers.py

```diff
@@ -521,17 +508,6 @@ def wait_for_all_resources_creation(pvc_count, pod_count, namespace, timeout=900
         sleep=5,
     )
 
-    # WA for Bug 2007376, Scale RBD mirror daemon deployment back to 1
-    rbd_mirror_deployment = get_deployments_having_label(
-        constants.RBD_MIRROR_APP_LABEL, constants.OPENSHIFT_STORAGE_NAMESPACE
-    )[0]
-    if rbd_mirror_deployment.replicas == 0:
-        logger.info(
-            f"Scaling up RBD mirror daemon deployment {rbd_mirror_deployment.name} back to 1"
-        )
-        rbd_mirror_deployment.scale(replicas=1)
-        rbd_mirror_deployment.wait_for_available_replicas()
```

Comment:
Shall we remove the function `wait_for_available_replicas()` as well? Or is it needed?

Reply:
It's already in removed list